tests/phpunit/integration/Core/Tags/First_Party_Mode/REST_First_Party_Mode_ControllerTest.php

```diff
@@ -239,7 +239,7 @@ public function test_get_fpm_server_requirement_status( $data ) {
 		// of the `is_endpoint_healthy()` method. The rest of the controller's behaviour remains unmocked.
 		$mock_controller = $this->getMockBuilder( First_Party_Mode::class )
 			->setConstructorArgs( array( $this->context ) )
-			->onlyMethods( array( 'is_endpoint_healthy' ) )
```

Comment:
`onlyMethods` isn't available on older versions of PHPUnit (or apparently polyfilled by the Yoast library)